ansible/doc/README.testbed.Overview.md

```diff
@@ -0,0 +1,56 @@
+# SONiC Testbed Overview
+
+This document gives overview of the SONiC testbed. 
+
+## Physical topology
+
+![](img/testbed.png)
+
+1. Every DUT port is connected to one of leaf fanout switches
+2. Every leaf fanout switch has unique vlan tag for every DUT port
+3. Root fanout switch connects leaf fanout switches and testbed servers
+4. Connections from root fanout switches are 802.1Q trunks
+5. Any testbed server can access any DUT port by sending a packet with the port vlan tag (root fanout switch should have this vlan number enabled on the server trunk)
+
+## Fanout switch
+*Fanout switch*: A physical switch which enables VLAN trunking. 
+   * Hardware SKU: Arista 7260 or similar
+   * Et33 is a vlan trunking port and is connected to the eth0 port of the linux host.
+   * Et1-Et32 are vlan access ports and are connect to DUT.
+   * Enable LACP/LLDP passthrough
+   * Disable spanning tree protocol
+
+## Testbed server
+
+- Server SKU (this is what we are using, not mandatory): Dell 730; 2 CPUs each has 18 cores; 192G memory; hard disk:2X500G
+- NIC: Mellanox MT27700 Family CX4
+
+![](img/testbed-server.png)
+
+### Network connections
+
+- The testbed server has 2 network ports:
+  - Trunk port (Mellanox MT27700 Family CX4) to root fanout switch
+  - Server management port to manage the server, VMs and PTF containers on the server
+ 
+### VMs
+
+The VMs use Arista vEOS. They are using to setup protocol test such as BGP, LACP, LLDP. They are created using ```testbed-cli.sh start-vms``` command. Each VM uses 2G of RAM and has 10 network interfaces:
+
+1. 8 front panel ports. These ports are connected to openvswitch bridges, which are connected to vlan interfaces. The vlan interfaces are connected to the fanout switch (through physical port).
+2. 1 back panel port. All testbed VMs connected to each other using this port (it isn't shown on the figure above).
```

Comment:
not all VMs are connection through the backplane.
Each topology has its own backplane